src/main/java/co/pragra/attendancesystem/controller/CohortController.java

```diff
@@ -0,0 +1,75 @@
+package co.pragra.attendancesystem.controller;
```

Comment:
Pls. refactor to rename this to CohortApi.java and move to co.pragra.attendancesystem.rest

CohortController.java will be used by the Cohort UI build.